src/main/frontend/handler/events.cljs

```diff
@@ -47,6 +51,14 @@
     close-fn)))
 
 (defmethod handle :graph/added [[_ repo]]
+  ;; add ast/version to db
+  (let [conn (conn/get-conn repo false)
+        ast-version (d/datoms @conn :aevt :ast/version)]
+    (when ast-version
```

Comment:
L57 ~ L59 can be replaced with `frontend.db/set-key-value`

Reply:
updated!